docs/release/releases/release-1.6.md

```diff
@@ -0,0 +1,34 @@
+# Cluster API v1.6
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.6` release cycle.
+
+| **What**                                             | **Who**      | **When**                   | **Week** |
+|------------------------------------------------------|--------------|----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 31st July 2023      | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 4th August 2023      | week 1   |
+| Team finalized                                       | Release Lead | Friday 4th August 2023      | week 1   |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 29th August 2023       | week 5   |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 26th September 2023      | week 9   |
+| v1.6.0-beta.0 released                               | Release Lead | Tuesday 24th October 2023     | week 13  |
+| Communicate beta to providers                        | Comms Manager| Tuesday 24th October 2023     | week 13  |
+| *v1.4.x & v1.5.x released*                           | Release Lead | Tuesday 24th October 2023     | week 13  |
+| v1.6.0-beta.x released                               | Release Lead | Tuesday 31st October 2023      | week 14  |
+| release-1.6 branch created (**Begin [Code Freeze]**) | Release Lead | Tuesday 7th November 2023     | week 15  |
+| v1.6.0-rc.0 released                                 | Release Lead | Tuesday 7th November 2023     | week 15  |
+| release-1.6 jobs created                             | CI Manager   | Tuesday 7th November 2023     | week 15  |
```

Comment:
I wonder if we want to move this one week back (and everything below as well). Given the year has more than 51 weeks we could make this release cycle 18 weeks.

The reason is that KubeCon NA is from 6-9 November

(cc @fabriziopandini)

Reply:
(also Thanksgiving is on 23rd November)

Reply:
> Given the year has more than 51 weeks we could make this release cycle 18 weeks.

Sounds like a good idea to me.

Reply:
@fabriziopandini Just for my understanding this means we would leave release-1.6 branch created, v1.6.0-rc.0 released     and release-1.6 jobs created  in the KubeCon week?

Reply:
We can also do nothing (no beta, no rc) during the KubeCon week and restart right after KubeCon, no strong preferences.

Reply:
I've updated the schedule with a call out for KubeCon idle week. Since this is a preliminary schedule I'll make sure to bring with up with the next Release Lead in case they want to modify anything before we finalize to it on August 4th.

Reply:
Sounds good!